konch.py

```diff
@@ -954,6 +955,25 @@ def preview_unauthorized() -> None:
     return None
 
 
+def __ensure_global_cfg_loaded(mod: typing.Union[types.ModuleType, None]) -> None:
+    """Load global config if not already loaded."""
+    try:
+        if (
+            globals()["_cfg"] is None
```

Comment:
I'm not sure all three of these checks are necessary.

Reply:
Agreed, assuming one or two of the states could not occur... but all three states are distinct, non-equivalent, and if they did occur, should be handled similarly.